app/Models/UserActivity.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class UserActivity extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'user_id',
+        'browser_name',
+        'platform',
+        'device',
+        'ip',
```

Comment:
Although this will typically be running on a local network, and no public IP's will be exposed we will probably want to:
```php
 protected $hidden = [
        'ip',
    ];
 ```